app/lib/rocketchat.js

```diff
@@ -1166,6 +1166,10 @@ const RocketChat = {
 		return !isUnread;
 	},
 
+	isSwipeEnabled(item) {
+		return !(item?.search || item?.joinCodeRequired || item?.outside);
```

Comment:
Where did we get this logic from?

Reply:
This logic I figured out from the return of the 'spotlight'